### ISSUE TITLE
MomentUI fails to display the title if an alt text is defined

### ISSUE THREAD

**Issue by lapineige** [pixelfed/pixelfed#2154]
I've finally found out how this issue https://github.com/pixelfed/pixelfed/issues/1728#issuecomment-601846873 https://github.com/YunoHost-Apps/pixelfed_ynh/issues/86 happens: when using MomentUI, if I publish an image with media description, the title is empty ("Untitled post").
Yet the title is displayed correctly in Mastodon, and in MetroUI too I think. And that "Oops, something went wrong" message starts to pop-up.

This doesn't seem to be Yunohost related, or at least I don't see in which way it could be the case… I don't know of the UI code is working. (Could it be linked to an horizon task issue ?)

**Comment by dansup** [comment]
Fixed in latest commits!

**Comment by lapineige** [comment]
https://github.com/pixelfed/pixelfed/commit/90b89cb8 and https://github.com/pixelfed/pixelfed/commit/59dd39b87d55e40f7e0d28167587f65f9cc20101 I guess.

That's good news, thanks :)

Is it planned for the next release ?
